code/espurna/config/hardware.h

```diff
@@ -112,6 +112,9 @@
     #define I2C_SDA_PIN         12  // D6
     #define I2C_SCL_PIN         14  // D5
 
+    #define SONAR_TRIGGER       12  // D6
+    #define SONAR_ECHO          13  // D7
+
```

Comment:
Maybe just keep as defaults in sensor.h?

Reply:
Agree, these should not be here.

Reply:
Oops. Pins from wemos do not include hw.h 7 and 8 (6-11 are not mapped to pins at all so they do nothing): https://github.com/esp8266/Arduino/blob/master/variants/d1/pins_arduino.h
What i meant to say, change the default to these and just not define it only for wemos hw :) Sry for confusion

Reply:
@xoseperez I don't think these will work on Wemos — https://github.com/xoseperez/espurna/blob/dev/code/espurna/config/hardware.h#L2669-L2671. Wasn't this change correct after all?

Reply:
@ruimarinho nah you linked travis-only hw that is only a built test, no real boards involved. these are fine https://github.com/xoseperez/espurna/blob/4f61d3aa7e2e6526fdb518a9f84785b055ac0ab7/code/espurna/config/sensors.h#L570-L576